iree/hal/cuda/dynamic_cupti_test.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/base/api.h"
+#include "iree/hal/cuda/dynamic_symbols.h"
+#include "iree/testing/gtest.h"
+
+namespace iree {
+namespace hal {
+namespace cuda {
+namespace {
+
+#define CUDE_CHECK_ERRORS(expr)           \
+  {                                       \
+    CUresult status = expr;               \
+    IREE_ASSERT_EQ(CUDA_SUCCESS, status); \
+  }
+
+#define CUPTI_CHECK_ERRORS(expr)           \
+  {                                        \
+    CUptiResult status = expr;             \
+    IREE_ASSERT_EQ(CUPTI_SUCCESS, status); \
+  }
+
+// Buffer request callack for CUPTI activity records.
+void bufferRequested(uint8_t **buffer, size_t *size, size_t *maxNumRecords) {}
```

Comment:
nit: runtime code follows Google C++ style and not LLVM C++ style, move the asterisks over